docs/tutorial/using-multiple-themes-together.md

```diff
@@ -2,4 +2,247 @@
 title: Using Multiple Themes Together
 ---
 
-_This tutorial is coming soon. In the meantime, check out the [introduction to using multiple themes in the docs](/docs/themes/using-multiple-gatsby-themes)._
+## What’s contained in this tutorial?
+
+By the end of this tutorial, you’ll have done the following:
+
+- learned how to compose multiple Gatsby themes into a final site
+- learned about component shadowing
+- learned about shadowing a Theme-UI theme file
+
+The themes we will be composing in this tutorial are [gatsby-theme-blog](https://www.gatsbyjs.org/packages/gatsby-theme-blog/), [gatsby-theme-notes](https://www.gatsbyjs.org/packages/gatsby-theme-notes/) and [gatsby-mdx-embed](https://gatsby-mdx-embed.netlify.com/). Follow along as we go from 'hello-world' to a fully functioning final site!
```

Comment:
```suggestion
The themes we will be composing in this tutorial are [gatsby-theme-blog](/packages/gatsby-theme-blog/), [gatsby-theme-notes](/packages/gatsby-theme-notes/) and [gatsby-mdx-embed](/packages/@pauliescanlon/gatsby-mdx-embed/). Follow along as we go from 'hello-world' to a fully functioning final site!
```
Inside docs we can use relative links (/packages/gatsby-theme-blog/ instead of https://www.gatsbyjs.org/packages/gatsby-theme-notes/), which utilize the Gatsby Link component

Reply:
Done, I think I caught them all!